reinforcement_learning/rl_mujoco_ray/rl_hopper_ray_gymEnv.ipynb

```diff
@@ -0,0 +1,534 @@
+{
```

Comment:
<p>Grammar - <code>from you local machine</code> -&gt; <code>from your local machine</code></p><p> </p><p>Typo: <code>dependenceis</code> -&gt; <code>dependencies</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3174/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>